Pedro

prdj opened the pull request.

prdj committed:
pedro

prdj committed:
pedro

Ungoot commented on pedro/index.css:
```diff
@@ -0,0 +1,89 @@
+    height: 100vh;
+    background-attachment:fixed;
+    background-color:#000000;
+    background-image: linear-gradient(to right, rgba(255,153,153,0) 0%,rgba(255,153,153,0) 70%,rgb(0, 0, 0) 100%), linear-gradient(to left, rgba(255,153,153,0) 0%,rgba(255,153,153,0) 20%,rgb(0, 0, 0) 100%), url(/images/background.jpg);
```
linear-gradient here is not necessary as it's not shown.

Ungoot commented on pedro/index.css:
```diff
@@ -0,0 +1,89 @@
+    margin: 15px 20%;       
+}
+
+form > div {
```
these 2 `position: relative` does not change the positioning because you did not assign an offset with `top` `left` etc. So they are unnecessary.

Ungoot commented on pedro/index.css:
```diff
@@ -0,0 +1,89 @@
+}
+
+.box2 {
+    position: stick;
```
There is a typo here it should be `position: sticky`

prdj committed:
Update index.css

Hi Bleda. I did the changes you sugested unless the on you refer to the linear-gradient. I do think it matters for the fade the pic does to the black background, anyway. I did the changes on github web because I dont have conction with anymore with this old repositoris.

Ungoot committed:
Merge pull request #9 from prdj/patch-1

Update index.css

Ungoot commented:
Good job, merged.
